src/docs/product/issues/suspect-commits/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Suspect Commits
+sidebar_order: 2
+redirect_from:
+  - /product/releases/suspect-commits/
+description: "Learn how integrations enable suspect commits."
+---
+
+## Suspect Commits & Suggested Assignees
+
+When you've enabled suspect commits, we can tie together the commits in the release, including:
+
+- Files touched by those commits
+- Files observed in the stack trace
+- Authors of those files
+- The issue number in a commit message
+
+Suspect commits and suggested assignees will then display on the **Issue Details** page in [sentry.io](https://sentry.io):
+
+![The suspect commits and suggested assignees for an issue](suspect-commits-highlighted.png)
+
+## Enable Suspect Commits
+
+This process assumes:
+
+- Source maps — or your platform-specific files for mapping transformed source code to the original source — have been uploaded (check out the docs for your [specific platform](/platforms/))
+
+#### Connect a Repository via integrations
+
+Suspect commits can be enabled via the following integrations:
+- GitHub
+- GitLab
+
+Head to your organization’s **Settings > Integrations** page, and click on the integration you’d like to use. You’ll need to configure the integration first, if you haven’t already and add the repository you'd like to use.
+
+#### Setup Code Mappings
```

Comment:
Is this section describing how you connect a repo via integration or is connecting step 1 and setting up code mappings step 2?

Reply:
you need to:
1. Connect the integration and grant access to your github repo / github org
2. Add the repo in sentry
3. Add a code mapping that links your sentry project to the repo